README.md

```diff
@@ -108,6 +108,27 @@ $ cat .scuttle.json
 # sctl run helmfile diff
 ```
 
+### Rotate state / re-key
+
+As you deprecate/disable older KMS key revisions, it can be prudent to migrate
+a statefile of encrypted secrets. This can be a daunting task if undertaken one
+by one. To ease the migration between versions (and different keys) the re-key
+function was introduced in 1.0.0-rc8.
```

Comment:
this says `1.0.0-rc8`, but the updated version here is `1.0.0` -- just double checking to see if this is right or if it should say 1.0.0 here

Reply:
yeah, the rc8 wont have an identifier if we cut 1.0.0 from this. I'll bump, good catch.